pkg/operator/controllers/machinehealthcheck/staticresources/machinehealthcheck.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineHealthCheck
+metadata:
+  name: openshift-machinehealthcheck 
```

Comment:
I would go with the name `aro-machinehealthcheck` rather than `openshift-machinehealthcheck` so it's clear that it's specific to our service.